root/etc/e-smith/templates/etc/ssh/sshd_config/45AllowGroups2Sshd

```diff
@@ -0,0 +1,54 @@
+{
+    # Allow groups to sftp/ssh with different policies
+    # legacy: everybody with a ssh account can login
+    # admins: groups of $admins{'group'} can login
+    # custom:  group listed $ssh{'AllowGroups'} can login
+    # members restricted to sftp (SftpRestrictedGroups) must be allowed to sshd also
+
+    use strict;
+    use esmith::ConfigDB;
+    use esmith::AccountsDB;
+    my $db = esmith::ConfigDB->open_ro();
+
+
+    sub uniq {
+        my %seen;
+        return grep { !$seen{$_}++ } @_;
+    }
+
+    my $domain = $db->get_value('DomainName')|| die ('Cannot retrieve DomainName');
```

Comment:
You could also use the already existing variable `$DomainName`

Reply:
thank but I prefer to be consistant, if I use the $key{prop}, I would prefer to use it everywhere, I recall also when you use the esmith library that the default configuration hashes could be broken

Reply:
For that matter, I could have seen also the alternative `$DB{'DomainName'}` syntax somewhere...

Reply:
tested it complains :D